app/code/Magento/Bundle/Test/Mftf/Test/NewProductsListWidgetBundleProductTest.xml

```diff
@@ -36,8 +36,10 @@
         <!-- Create a product to appear in the widget, fill in basic info first -->
 
         <actionGroup ref="AdminOpenProductIndexPageActionGroup" stepKey="amOnProductList"/>
-        <click selector="{{AdminProductGridActionSection.addProductToggle}}" stepKey="clickAddProductToggle"/>
-        <click selector="{{AdminProductGridActionSection.addBundleProduct}}" stepKey="clickAddBundleProduct"/>
+        <comment userInput="StepKey Backward Compatibility" stepKey="clickAddProductToggle"/>
```

Comment:
That will do the trick. However, it might not be obvious for me as a developer why we add this comment according to the current description. I would recommend putting some meaningful description like 

```
userInput="Adding the comment to replace clickAddProductToggle action for preserving Backward Compatibility"
```

It applies to all other comments in all your PRs. 

Thanks

Reply:
Hi @rogyar
I have applied your recommendations
Thanks :)